src/views/Correction/FmCorrection.vue

```diff
@@ -137,6 +137,9 @@ export default class FmCorrection extends Mixins(CommonMixin, DateMixin, FilingT
       // initialize Fee Summary data
       this.setFilingData(this.correctionResource.filingData)
 
+      // initialize the No Fee as pre-selected
+      this.setStaffPayment({ option: StaffPaymentOptions.NO_FEE })
```

Comment:
Why not just set this option in the firm resource files that are use in the line above to initialize the Fee Summary data?

Reply:
The firm resource files allows you to set the fee but I don't think that will set the no fee option in the staff payment component...

Reply:
Looks like current objects/component don't allow this directly.

I approve the original code.